sparv/core/io.py

```diff
@@ -91,8 +91,14 @@ def _write_single_annotation(
     if is_span:
         if not isinstance(values, list):
             values = list(values)
-        # Make sure that spans are sorted
-        assert all(values[i] <= values[i + 1] for i in range(len(values) - 1)), "Annotation spans must be sorted."
+        # Validate that spans are sorted
+        for i in range(len(values) - 1):
+            if values[i] > values[i + 1]:
+                raise RuntimeError(
```

Comment:
Commit message mentions `SparvErrorMessage` but you are using `RuntimeError`, which doesn't work since "TypeError: RuntimeError() takes no keyword arguments".

Reply:
Of course it should be SparvErrorMessage, fixing it